client/src/modules/accounts/modals/import.html

```diff
@@ -0,0 +1,97 @@
+<form name="ActionForm" bh-submit="ImportAccountsCtrl.submit(ActionForm)" novalidate>
+  <div class="modal-header">
+    <ol class="headercrumb">
+      <li class="static" translate>FORM.LABELS.ACCOUNT</li>
+      <li class="title text-capitalize" translate>
+        ACCOUNT.IMPORT.ACCOUNTS
+      </li>
+    </ol>
+  </div>
+  <div class="modal-body">
+    <!-- option -->
+    <div class="form-group">
+      <div class="radio">
+        <label>
+          <input type="radio" name="importingOption" id="default_ohada_accounts_option" 
+            ng-model="ImportAccountsCtrl.option" 
+            ng-value="1" 
+            ng-change="ImportAccountsCtrl.onChange(ImportAccountsCtrl.option)">
+          <span translate>ACCOUNT.IMPORT.LOAD_DEFAULT_OHADA_ACCOUNTS</span>
+        </label>
+      </div>
+      <div class="radio">
+        <label>
+          <input type="radio" name="importingOption" id="import_ohada_accounts_option" 
+            ng-model="ImportAccountsCtrl.option" 
+            ng-value="2"
```

Comment:
To make this a bit more readable, it would be nice to make these numbers into variables attached to the controller.  That way, we can give them a name.  Like this:
```js
vm.LOAD_OHADA_ACCOUNTS = 2
```
then you can use it in your HTML like:
```html
<input type="radio" name="importingOption" id="import_ohada_accounts_option" 
            ng-model="ImportAccountsCtrl.option" 
            ng-value="ImportAccountsCtrl.LOAD_OHADA_ACCOUNTS">
```

It will just make it easier to read.